src/network/http/websocket.cpp

```diff
@@ -679,7 +672,7 @@ namespace fc { namespace http {
 
 
 
-   websocket_client::websocket_client( const std::string& ca_filename ):my( new detail::websocket_client_impl() ),smy(new detail::websocket_tls_client_impl( ca_filename )) {}
+   websocket_client::websocket_client( const std::string& ca_filename):my( new detail::websocket_client_impl() ),smy(new detail::websocket_tls_client_impl( ca_filename )) {}
```

Comment:
Whitespace changes should at least be consistent. Better don't change at all if it's whitespace-only. Only clutters up the diff.

Reply:
Reverted to the way it was.